README.adoc

```diff
@@ -4,6 +4,21 @@
 :toc-title:
 :toclevels:
 
+[NOTE]
+====
+Each released `crc` binary expires 30 days after the release due to certificates embedded in the Openshift cluster.
+Using an older `crc` binary will result in an expired certificates error.
+Following steps are needed to resolve this error.
```

Comment:
```adoc
To resolve this error:

* link:https://github.com/code-ready/crc/releases[Download the latest CodeReady Containers release] and place the `crc` binary in your `$PATH`.
* Remove the cluster with certificate errors using the `crc delete` command.
* Set up the new release with the `crc setup` command.
* Start the new cluster with the `crc start` command.
```

I'm not sure that we have to be so specific regarding "and place the `crc` binary in your `$PATH`", but the rest of the suggestion still stands. :)